src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -353,6 +392,12 @@ static HttpPortRange ParsePortRangeOption(ArgumentResult result)
             }
         }
 
+        private static void UseQuitCommand(IDisposable disposeOnQuit)
+        {
+            Quit.DisposeOnQuit = disposeOnQuit;
+            KernelCommandEnvelope.RegisterCommandType<Quit>(nameof(Quit));
```

Comment:
Doing this without a static would be preferable.

It seems like adding the command to the kernel, including the implementation, could avoid this, e.g.

```csharp
kernel.UseCommand<Quit>();
```

Reply:
liked it and did something along that line